apps/full-site-editing/package.json

```
@@ -16,10 +16,10 @@
 	},
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
-		"plugin": "webpack --source='plugin'",
+		"plugin": "cb --source='plugin' --env.WP",
 		"dev:plugin": "npm run plugin",
 		"build:plugin": "NODE_ENV=production npm run plugin",
-		"theme": "webpack --source='theme'",
+		"theme": "cb --source='theme' --env.WP",
```

Already hard coded:

https://github.com/Automattic/wp-calypso/blob/acfc8a24bb929b72ef109e7d1d0e3ef7ed6eaafb/apps/full-site-editing/webpack.config.js#L32

```suggestion
		"theme": "cb --source='theme'",
```